doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+# Practical Overview of Explainability
+
+While the applications of machine learning are impressive many models provide predictions that are hard to interpret or reason about. This limits their use in many cases as often we want to know why and not just what a model made a prediction. Alarming predictions are rarely taken at face value and typically warrant further analysis. Indeed, in some
```

Comment:
Sentence `This limits...` needs to be rewritten.

Also, I would suggest bolding **why** and **what** for emphasis.